.env

```diff
@@ -0,0 +1,3 @@
+ REACT_APP_API_KEY=5402e464f97c44968342dd3057b4a262
```

Comment:
![image](https://github.com/milos135/reactNFTWithMetamask/assets/85948457/6d2b3530-40ff-4262-8159-7045e978e9f3)
place pagination below cards

Reply:
i am viewing this on 4k resolution , its wraping on your monitor prolly

Reply:
and Milos, for the third time, unify the FONT on your app each and every letter should use the same font

Reply:
I add a font in index.css file, and check pagination, on every device is now the same position